AzureMonitorAgent/ama_tst/modules/logcollector.py

```diff
@@ -234,6 +234,18 @@ def create_outfile(output_dirpath, logs_date, pkg_manager):
             outfile.write("--------------------------------------------------------------------------------\n")
         outfile.write("--------------------------------------------------------------------------------\n")
 
+        # du output on events folder
+        for flag in ["", "--apparent-size"]:
+            du_full_cmd = DU_CMD.format(flag)
+            outfile.write("Output of command: {0}\n".format(du_full_cmd))
+            outfile.write("========================================\n")
+            outfile.write(helpers.run_cmd_output(du_full_cmd))
```

Comment:
Do we need to append a newline after the cmd output?

Reply:
The `run_cmd_output()` function automatically adds a newline already